components/GettingStarted.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from 'react';
+import { atomOneDark } from 'react-syntax-highlighter/dist/cjs/styles/hljs';
+import Highlight from 'react-syntax-highlighter';
+
+async function fetchData() {
+  const response = await fetch('/data/getting-started-examples.json');
+  const data = await response.json();
+  return data;
+}
+
+async function fetchSchemaData() {
+  const response = await fetch(
+    '/data/getting-started-examples/schemas/default.json',
+  );
+  const schemaData = await response.json();
+  return schemaData;
+}
+
+async function fetchedInstanceData() {
+  const response = await fetch(
+    '/data/getting-started-examples/instances/default-ok.json',
+  );
+  const schemaData = await response.json();
+  return schemaData;
+}
+
+const GettingStarted = () => {
+  const [options, setOptions] = useState([]);
+  const [instances, setInstances] = useState([
+    {
+      name: 'Valid instance',
+      default: true,
+      valid: true,
+      file: '/data/getting-started-examples/instances/default-ok.json',
+      details: 'This is a valid JSON instance for the provided JSON Schema',
+    },
+    {
+      name: 'Invalid instance',
+      default: false,
+      valid: false,
+      file: '/data/getting-started-examples/instances/default-ko.json',
+      details: 'This is an invalid JSON instance for the provided JSON Schema',
+    },
+  ]);
+
+  const [details, setDetails] = useState([
+    'This is a valid JSON instance for the provided JSON Schema',
+    true,
+  ]);
```

Comment:
Hi Dhairya, all those methods should disappear from here as they are overriding the information available in the json files. The idea is this component to read getting-started-examples.json and find all this information.

Probably this is here because you was testing something, but this should disappear and the behaviour should be totally data driven with getting-started-examples.json as source.

Reply:
Hi Benja I have kept this data here to initialize the state variables, if they are removed it will further affect other state variables and its functioning